converters/fromZigbee.js

```diff
@@ -2595,6 +2595,26 @@ const converters = {
             return result;
         },
     },
+    ts0219_siren: {
+        cluster: 'ssIasZone',
+        type: 'attributeReport',
+        convert: (model, msg, publish, options, meta) => {
+            const zoneStatus = msg.data.zoneStatus;
+            return {
+                alarm: (zoneStatus & 1) > 0,
```

Comment:
If this works, can you try to use `fz.ias_siren` instead of `fz.ts0219_siren`

Reply:
ias_siren return a lot of wrong / bad datas : 
tamper : no tamper detection on this device;
battery_low : is never passed to true even if i wait the battery die
supervision_reports: i don't know what it is
restore_reports : i don't know what it is
ac_status: is not good, that's why i use the ac_connected: powerSource === 2 ? true : false,
test: i don't know what it is

Can i use ias_siren only for "alarm" and prevent all other datas to be returned ?

Reply:
I see, will make a generic converter of this